apps/frontend/src/pages/root/components/user-wheel/libs/components/scores-edit-modal/scores-edit-modal.tsx

```diff
@@ -41,8 +41,14 @@ const ScoresEditModal: React.FC<Properties> = ({
 					item.categoryId === categoryId ? { ...item, score: value } : item,
 				),
 			);
+
+			const changedScores = scores.map((score) =>
+				score.categoryId === categoryId ? { ...score, score: value } : score,
+			);
+			void dispatch(quizActions.editScores({ items: changedScores }));
```

Comment:
We shouldn't save scores immediately. We save when the user clicks on a button, now we just update the wheel